modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -260,7 +258,7 @@ object SeqexecEngine {
                       odbQueuePollingInterval: Duration)
   val defaultSettings = Settings(Site.GS,
     "localhost",
-    LocalDate.of(2017, 1,1),
+    LocalDate.of(2017, 1, 1),
```

Comment:
The idea was that the DHS simulator is passed the current date, so you can have labels like S20170614S0001 on June 14th, 2017. But if we read the current date inside the simulator, then you have a side effect. Actually, it was like that at some point, but was changed to its current form.